src/Fixer/Import/FullyQualifiedStrictTypesFixer.php

```diff
@@ -447,27 +453,30 @@ private function shortenClassIfPossible(Tokens $tokens, array $class, array &$us
             $this->registerSymbolForImport('class', $longTypeContent, $uses, $namespaceName);
         }
 
-        if (str_starts_with($longTypeContent, '\\')) {
-            $typeName = substr($longTypeContent, 1);
+        if (str_starts_with($longTypeContent, '\\') || '' === $namespaceName) {
+            $typeName = str_starts_with($longTypeContent, '\\') ? substr($longTypeContent, 1) : $longTypeContent;
```

Comment:
Wouldn't this be enough?

```suggestion
            $typeName = ltrim($longTypeContent, '\\');
```

Reply:
Why not, landed.